admin-manual/installation/installation.rst

```diff
@@ -192,9 +192,11 @@ Using 16.04 (Xenial):
 
 .. code:: bash
 
-   sudo wget -O - https://packages.archivematica.org/1.6.x/key.asc | sudo apt-key add -
-   sudo sh -c 'echo "deb [arch=amd64] http://jenkins-ci.archivematica.org/1.8.x/ubuntu xenial main" >> /etc/apt/sources.list'
-   sudo sh -c 'echo "deb [arch=amd64] http://packages.archivematica.org/1.6.x/ubuntu-externals trusty main" >> /etc/apt/sources.list'
+   sudo wget -O - http://jenkins-ci.archivematica.org/repos/devel.key | sudo apt-key add -
+   sudo wget -O - https://packages.archivematica.org/1.7.x/key.asc | sudo apt-key add -
```

Comment:
Can we move this key somewhere else so it's not under the 1.7.x path? This is going to be a source of confusion.

Reply:
Created a 1.8.x link pointing to 1.7.x, this way we don't need to duplicate efforts when we add a new externals package, and once 1.7.x is released, we can remove the symbolic link and continue as usual.